msg/vehicle_odometry.msg

```diff
@@ -62,5 +62,5 @@ float32 yawspeed		# Angular velocity about Z body axis
 # If angular velocity covariance invalid/unknown, 16th cell is NaN
 float32[21] velocity_covariance
 
-# TOPICS vehicle_odometry vehicle_mocap_odometry vehicle_visual_odometry vehicle_visual_odometry_aligned
+# TOPICS vehicle_odometry vehicle_mocap_odometry vehicle_visual_odometry
```

Comment:
We are removing this but we keep it as an alias in `uorb_rtps_message_ids.yaml`? I am not sure what will be the result here

Reply:
The `uorb_rtps_message_ids.yaml` alias seems to be in place for most of the cases of multiple topics per uORB message (type). What's the intended purpose?

Reply:
It also means that an alias shares the same message structure as the base message, i.e. `vehicle_visual_odometry_aligned` shares the same structure as `vehicle_odometry`.

Reply:
But I guess this works, if the code generation doesn't complain.